lib/housekeeping.js

```diff
@@ -38,9 +38,24 @@ class HouseKeeper {
     this.tagger = tagger;
   }
 
+  async handleVolumeData(volume) {}
+
+  async sweepVolumes() {
+    let volumes = await this.runaws(this.ec2[region], 'describeVolumes', {
+      Filters: [{
+        Name: 'status',
+        Values: ['available', 'in-use'],
+      }],
+    });
+
+    for (let volume of volumes.Volumes) {
+      handleVolumeData(volume);
```

Comment:
handleVolumeData is async, and needs to be awaited.

Reply:
Also, if we're doing anything potentially time-consuming in `handleVolumeData`, then we should probably call it in parallel for each returned volume:
```
await Promise.all(volumes.Volumes.map(handleVolumeData));
```
This calls `handleVolumeData` for each element of the array, then collects all of the resulting Promises and waits until they are all complete (or one fails).

Reply:
I would vote for the `Promise.all` approach anyway. Easier to read, easier to write and more effient.